tests/testthat/test-mathjax.R

```diff
@@ -0,0 +1,41 @@
+test_that("adding mathjax works", {
+  withr::local_options(usethis.quiet = TRUE)
+  temp_dir <- withr::local_tempdir()
+  proj <- usethis::create_package(
+    file.path(temp_dir, "testPackage"),
+    rstudio = FALSE,
+    open = FALSE
+  )
+  usethis::local_project(proj)
+  usethis::use_readme_md()
+  usethis::use_pkgdown()
+  yaml <- yaml::read_yaml("_pkgdown.yml")
+  yaml$mathjax <- TRUE
+  yaml::write_yaml(yaml, "_pkgdown.yml")
+  expect_output({
```

Comment:
this is just to avoid the messages printed by pkgdown. (no way to toggle them for now if I follow correctly)